mlir/lib/Dialect/Rock/IR/WmmaInsnGroup.cpp

```diff
@@ -84,6 +84,10 @@ FailureOr<WmmaInsn> WmmaInsn::select(mlir::Type elementTypeA,
     insn = ROCDL::wmma_f32_16x16x16_bf16::getOperationName();
   } else if (elementTypeA.isInteger(8)) {
     insn = ROCDL::wmma_i32_16x16x16_iu8::getOperationName();
+  } else if (elementTypeA.isFloat8E4M3FN()) {
+    // insn = ROCDL::wmma_
```

Comment:
Comments?

Reply:
Sorry, somehow this was overridden. Should be fine now